dotnet/src/Connectors/Connectors.Memory.AzureCognitiveSearch/AzureCognitiveSearchMemoryStore.cs

```diff
@@ -428,5 +432,17 @@ private static T RunMemoryStoreOperation<T>(Func<T> operation)
         }
     }
 
+    private static double ScoreToCosineSimilarity(double score)
```

Comment:
1. Please add XML comment to the method and the one below.
2. Rename the method and the one below so that the start from verb - https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-methods

Reply:
private methods are rarely commented and the name is self explanatory. About the verb, this is a transformation like `ToInt` `ToString`